spec/unit/xml_params_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+describe XmlParams do
+  source_filepath = './tmp/86476809/(NS)-CHAUZIMU-MWA-CHILENGEDWE.doc'
```

Comment:
For unit tests I think we can safely remove those complicated values and replace them with something like

```
source_filepath = '/source/file.doc
```

And similar to `converted_filepath`